src/Data/Array/Polarized/Pull.hs

```diff
@@ -42,11 +106,12 @@ import qualified Unsafe.Linear as Unsafe
 asList :: Array a #-> [a]
 asList = foldr (\x xs -> x:xs) []
 
--- | /!\ Partial! Only works if both arrays have the same length.
+-- | @zipWith f [x1,x2,...] [y1,y2,...] == [f x1 y1, f x2 y2, ...]@
+-- __Partial:__ Only works if both arrays have the same length.
```

Comment:
```suggestion
-- | @zipWith f [x1,x2,...,xn] [y1,y2,...,yn] = [f x1 y1, f x2 y2, ..., f xn yn]@
-- __Partial:__ `zipWith f [x1,x2,...,xn] [y1,y2,...,yp]` is an error if @n ≠ p@
```
(avoid non-sentences, also, `==` is the symbol for equality test, we are interested here in true equality. For instance this equation still works if the `xi` are functions, whereas it wouldn't make sense to call `==`, so use the symbol `=` in its mathematical sense)